ee/clickhouse/queries/trends/normal.py

```diff
@@ -62,12 +62,11 @@ def _format_normal_query(self, entity: Entity, filter: Filter, team_id: int) ->
             agg_query = self._determine_single_aggregate_query(filter, entity)
```

Comment:
I think this function's queries are missing the new `date` and `data` notations which is causing the query to fail when table/pie is requested